syntax = "proto2";
package dhc;

//using index guid 0
//using load guid

message mail_server_t {
    optional uint64 guid                    = 1;
    repeated uint64 receiver_guids          = 2;
    repeated uint64 delete_guids            = 3;
    optional uint64 sender_date             = 4;
    optional string title                   = 5;
    optional string text                    = 6;
    optional string sender_name             = 7;
    repeated int32 type                     = 8;
    repeated int32 value1                   = 9;
    repeated int32 value2                   = 10;
    repeated int32 value3                   = 11;
    optional uint64 creat_time              = 12;
}
